frontend/src/lib/lemon-ui/LemonButton/LemonButton.tsx

```diff
@@ -26,7 +26,7 @@ export interface LemonButtonPropsBase
         | 'role'
         | 'aria-haspopup'
     > {
-    children?: React.ReactNode
+    children?: React.ReactNode | undefined
```

Comment:
weird why do we need `blah?: Type | undefined` on this line and not the ones below? I thought the `?` _was_implicitly `| undefined`

Reply:
Yeah, this actually came up in a different PR with type changes I was working on. It's not needed so removing